api/SVGNumberList.json

```diff
@@ -0,0 +1,52 @@
+{
+  "api": {
+    "SVGNumberList": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGNumberList",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": null
+          },
+          "chrome_android": {
+            "version_added": null
```

Comment:
All 3 of these should be marked `true`. https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/svg/svg_number_list.idl